back/lib/participation_method/ideation.rb

```diff
@@ -11,7 +11,7 @@ def additional_export_columns
     end
 
     def allowed_extra_field_input_types
-      %w[section number linear_scale text multiline_text select multiselect multiselect_image]
+      %w[section number linear_scale text multiline_text select multiselect multiselect_image ranking]
```

Comment:
Is this useful on ideation?

Reply:
@jamesspeake Yes, we're supporting both the ranking and matrix fields for ideation as well :)